website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,83 @@
 ---
 title: "Job notifications"
 id: "job-notifications"
-description: "Set up notifications in dbt Cloud to receive Email or Slack alerts for job run status."
+description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
+Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
 
-Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+## Email notifications
 
-### Email
+You can receive email alerts about jobs by configuring the dbt Cloud email notification settings.
 
-These are the following options for setting up email notifications. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for info on license types eligible for email notifications. 
+### Prerequisites 
+- You must be either a _developer user_ or an _account admin_ to configure email notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+    - As a developer user, you can set up email notifications for yourself. 
+    - As an account admin, you can set up notifications for yourself and other team members. 
 
-- As a **user** &mdash; You can set up email notifications for yourself under your Profile.
-- As an **admin** &mdash; You can set up notifications on behalf of your team members. 
+### Configure email notifications
 
-To set up job notifications, follow these steps:
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. By default, dbt Cloud sends notifications to the email address that's in your [User profile](https://cloud.getdbt.com/settings/profile) page.
```

Comment:
We should avoid product links like this in the docs. Multi-cell architecture is coming soon and will break these (also to avoid our docs alienating emea and AU clients since this feature is available for them as well).
```suggestion
1. By default, dbt Cloud sends notifications to the email address in your User profile page.
```

Reply:
good callout!